### ISSUE TITLE
Missing link text reported when generated via CSS pseudo-content

### ISSUE THREAD

**Issue by weboverhauls** [dequelabs/axe-core#880]
Missing link text is (mistakenly) reported when generated via CSS pseudo content. The link text appears visually and announces fine using NVDA+FFX and VO+Safari.

aXe reports:
Links must have discernible text
4.1.2.a Name, Role, Value

Page: 
https://www.saatvamattress.com/checkout/payment-options

HTML
`<a href="/checkout/payment-options" class="router-link-exact-active router-link-active visited" data-text-lg="payment options" data-text-sm="options"><span></span></a>`

CSS
```
.breadcrumb>div a:before {
    content: attr(data-text-lg);
}
```
Occurred with aXe Chrome Plugin:
axe-devtools 3.3.0
axe-core 3.0.0

![image](https://user-images.githubusercontent.com/473400/39651078-88c595a2-4fb8-11e8-9d6c-2434f6b02201.png)

**Comment by WilcoFiers** [comment]
We're aware of this. Axe-core currently doesn't take content of pseudo elements into consideration.

**Comment by WilcoFiers** [comment]
David MacDonald did some testing with pseudo elements. Turns out providing text content with just pseudo elements isn't supported by IE11:
http://davidmacd.com/blog/inserting-content-with-css-accessibility.html

This means that axe-core shouldn't pass these, as IE11 support is part of our [baseline](https://github.com/dequelabs/axe-core/blob/develop/doc/accessibility-supported.md). I'm going to close this issue for that reason.
